http/http-advanced-reactive/src/main/java/io/quarkus/ts/http/advanced/reactive/GrpcResource.java

```diff
@@ -20,6 +26,47 @@ public class GrpcResource {
     @GrpcClient("hello")
     Greeter client;
 
+    @Inject
+    @GrpcClient("reflection-service")
+    MutinyServerReflectionGrpc.MutinyServerReflectionStub reflection;
+
+    @GET
+    @Path("reflection/service/count")
+    @Produces(MediaType.TEXT_PLAIN)
+    public int serviceCount() {
+        ServerReflectionRequest request = ServerReflectionRequest.newBuilder().setHost("localhost")
+                .setListServices("").build();
+        GrpcReflectionResponse response = new GrpcReflectionResponse(invoke(request));
+        return response.getServiceCount();
+    }
+
+    @GET
+    @Path("reflection/service/list")
+    @Produces(MediaType.APPLICATION_JSON)
+    public List<String> serviceList() {
+        ServerReflectionRequest request = ServerReflectionRequest.newBuilder().setHost("localhost")
+                .setListServices("").build();
+        GrpcReflectionResponse response = new GrpcReflectionResponse(invoke(request));
+        return response.getServiceList();
+    }
+
+    @GET
+    @Path("reflection/service/description")
+    @Produces(MediaType.TEXT_PLAIN)
+    public String serviceMethods() {
+        ServerReflectionRequest request = ServerReflectionRequest.newBuilder()
+                .setHost("localhost").setFileByFilename("helloworld.proto").build();
```

Comment:
`helloworld.proto` and "" for `setListServices`. What about if you expose one endpoint that accepts nullable filename and returns `GrpcReflectionResponse`?